lib/node_modules/@stdlib/math/base/special/imuldw/docs/repl.txt

```diff
@@ -0,0 +1,30 @@
+
+{{alias}}( [out,] a, b )
+    Multiplies two unsigned 32-bit integers and returns an array of two
+    unsigned 32-bit integers which represents the 64-bit unsigned integer
+    product.
```

Comment:
```suggestion
```

You'll want to check that the lines are the proper width, but I believe the edits above should be correct.